### ISSUE TITLE
Not found repository

### ISSUE THREAD

**Issue by doctore** [wokwi/wokwi-cli#5]
Hi

Sorry to add an issue in the wrong repository but https://github.com/urish/angular-spinner was archived some weeks ago and it doesn't allow create new ones.

We are using different versions of this library (from **0.6.1** to **1.0.1**) in several projects, however invoking `bower install`, we receive the an error related with the source repository:

<img width="755" alt="Jenkins" src="https://github.com/wokwi/wokwi-cli/assets/6932389/3883d1ce-05f1-49ce-abeb-0d95070f23d2">

Taking a look to the generated `.bower.json`, we can see the configured options for the library:

![BowerFile](https://github.com/wokwi/wokwi-cli/assets/6932389/d595084f-a1d6-45f9-b8b3-492148fe6a52)

However, neither the source repository: https://github.com/Attrash-Islam/bower-angular-spinner.git nor the related Github account: https://github.com/Attrash-Islam are currently available.

Could you help us configuring a new available source on your side for this library?

Thanks in advance.

**Comment by urish** [comment]
Sorry, but I do not have anything newer than what's already on GitHub, and the reason that the repository is archived is because I don't have the bandwidth the answer queries regarding this old project.

**Comment by doctore** [comment]
I'm not asking anything related with the library, the problem is here is => its configured source was removed, so everybody using the library will have the same error.

**Comment by urish** [comment]
I understand, but as I explained, I do not have the bandwidth to deal with this.

**Comment by doctore** [comment]
ok I see. Thanks, I guess changing a URL is too time consuming.

**Comment by vandaribeiro** [comment]
I'm having the exact same problem. Did you manage to solve it @doctore ?

**Comment by doctore** [comment]
Finally we cloned the repository with the "real code" in the company's Bitbucker, so we could use it as an source, even modifying the `.bowerrc.json`, avoiding it doesn't use a no longer exists URL (GitHub account).

We have even gone one step further and installed the option to work with bower components in the company's Nexus, following the link:

https://help.sonatype.com/repomanager3/nexus-repository-administration/formats/bower- repositories

And all this, because someone is so fucking lazy and his time is so fucking precious to change a url's value.

**Comment by Vollbrecht** [comment]
> Finally we cloned the repository with the "real code" into the company's Bitbucker, so we could use it as an source, even modifying the `.bowerrc.json`, avoiding it to use a no longer exists URL (GitHub account).
> 
> We have even gone one step further and installed the option to work with bower components in the company's Nexus, following the link:
> 
> [https://help.sonatype.com/repomanager3/nexus-repository-administration/formats/bower- repositories](https://help.sonatype.com/repomanager3/nexus-repository-administration/formats/bower-repositories)
> 
> And all this, because author's time so fucking precious to change a URL's value.

Please stop being entitled about a open source project that you used. I tell you something, if you are a business owner and fail to asses such a simple thing, just shows that no one should ever do business with you for there own sake. Being rude to somebody on-top of it, is just the tip of the iceberg here.

**Comment by doctore** [comment]
If you aren't going to maintain a project, ok, I can understand it. But there are not sure how many people with the same problem reading "yes, but this is not my problem now and I cannot change a URL" and due to the owner update it to only readonly, no way to include issues in the right location.

I answered to vandaribeiro's question, if you consider I was so rude, deal with it, in the same way I had to do it with the owner's response.
